backend/src/main/resources/liquibase/create-tables-changelog-1.0.xml

```diff
@@ -1238,4 +1238,48 @@
                                  referencedColumnNames="id"
                                  referencedTableName="keyvalue"/>
     </changeSet>
+
+    <changeSet author="lineage" id="new_lineage">
+        <createTable tableName="lineage_id">
+            <column name="id" type="int8" autoIncrement="true">
```

Comment:
I'm not sure I understand what this `id` represents. It cannot be the same `id` that the protos have since that ID can be associated with multiple IO. One ID will represent a set of inputs and a set of outputs.

Reply:
it is confusing to me as well since this is a single column table with auto increment id

Reply:
Actually, nevermind. This was actually correct. Didn't noticed it was a single table.